.gitignore

```diff
@@ -69,3 +69,4 @@ ecr.repo
 
 #Helm
 **/Chart.lock
+/.trivyignore
```

Comment:
if you do this then there won't be a `.trivyignore` file when the trivy security job runs (as it doesn't run gradle)